### DISCUSSION TITLE
next/head is not working in nextjs 13

### DISCUSSION THREAD

**Discussion by francisdurairaj** [vercel/next.js#42099] [Help]
I am migrating the existing next js app to next 13. Where i am not able to add meta tag from pages other than from layout.

for example
app/Layout.tsx  

export default function RootLayout({
  children,
}: {
  children: React.ReactNode
}) {
  return (
    <html>
      <head>
        <title>my layout title</title>
        <meta name="viewport" content="initial-scale=1.0, width=device-width" />
      </head>
      <body>        
          {children}
      </body>
    </html>
  )
}

app/Page.tsx

export default function Page() {
    return (
        <div className='container'>
            <Head>
                <title>Home</title>                
                <meta name="description" content="home meta data" />                
            </Head>
            <>
                Home
            </>
        </div>
    )
}

where only "my layout title" is being rendered but the home title and meta is not working

**Comment by alvarlagerlof** [comment]
You need to use the new [head.js](https://beta.nextjs.org/docs/api-reference/file-conventions/head) file instead.
